contracts/common/StorageAccessible.sol

```diff
@@ -15,7 +15,7 @@ abstract contract StorageAccessible {
      * @return the bytes that were read.
      */
     function getStorageAt(uint256 offset, uint256 length) public view returns (bytes memory) {
-        bytes memory result = new bytes(length * 32);
```

Comment:
could you please add a similar comment here?

Reply:
Done 👍🏾